src/Illuminate/Http/Resources/Json/ResourceResponse.php

```diff
@@ -53,7 +53,7 @@ public function toResponse($request)
     /**
      * Wrap the given data if necessary.
      *
-     * @param  array  $data
+     * @param  array|Collection  $data
```

Comment:
Please use FQN and place the class first, thanks.

Reply:
Done